Delphi.WebMock.RequestStub.pas

```diff
@@ -117,6 +126,14 @@ function TWebMockRequestStub.WithBody(
   Result := Self;
 end;
 
+function TWebMockRequestStub.WithBody(const ADataName, ADataValue: string): TWebMockRequestStub;
+var LValue: string;
+begin
+  LValue := 'form-data; name="%s"' + sLineBreak + sLineBreak + '%s' + sLineBreak;
```

Comment:
Will this pattern work if the `content-disposition` of `form-data` isn't followed directly by the content field values? Also, would the pattern be more readable if it used RegEx sequences to match the line breaks?

Reply:
Dunno :) 
As I wrote earlier, for WebMocks to work I required the ability to distinguish Field Data.

Reply:
Another thing to consider is that **eventually** the Form data may be best properly parse instead of just text matched.

Reply:
I think you're probably right about the need to parse the body fully which will not only resolve the issues I've raised but will provide greater flexibility in matching. The resulting fluent syntax may also be more descriptive in it's design e.g. `WithBody(FormData.WithField('Field', 'Value'))` or something like that.

If you're in agreement then I'll close this PR and write up an issue as a future enhancement.

Reply:
Looks like a great solution.